examples/workflows/run_zeopp_workflow.py

```diff
@@ -0,0 +1,17 @@
+import os
+from aiida.common.example_helpers import test_and_get_code  # noqa
+from aiida.orm.data.base import Float
+from aiida.orm.data.cif import CifData
+from aiida.work.run import submit
+from aiida_zeopp.workflows import ZeoppGeometryWorkChain
+
+probe_radius = 1.8
+pwd = os.getcwd()
```

Comment:
your solution works only when you are in  `/examples/workflows` when running the workflow.
To generalize:
```suggestion
pwd = os.path.dirname(os.path.realpath(__file__))
```